### ISSUE TITLE
Hardpoint random points

### ISSUE THREAD

**Issue by Haox09** [deferred-impact/battledudes-issue-tracker#114]
In hardpoint if you are a long time player you can already predict where the next point will be. This have a major advantage from the enemy newbies. Long time players can get to the point quicker. My idea is that in every hardpoint round the capturing point will be random every round. Hardpoint should also use new point locations for example on riverside the buildings on the left side of the bridge on the north side rarely gets used. So is some buildings in westland. I think hardpoint should use them. Control points, however, should remain as it is because it's 3 points and not one.

**Comment by deferred-impact** [comment]
Well, one could argue that it's part of the skill. Learning the points sequence, learning the time at which points rotate, keeping track of the timer, and then actually utilizing that minor (because, honestly, it's rather minor unless *your* entire team does it and the entire *enemy* team does not) advantage into a meaningful profit.

That said, i do see some appeal in a randomized Hardpoint. It would be less strategic and more about fighting, sure, but that's not necessarily bad.

Ideally, this would be an optional change, i.e. some sort of voting option. Either a separate voting option ("points random" vs "points predictable" (although, how would that work if a different gamemode is chosen?)), or simply an extra version of gamemode ("Westlands with vehicles"?  "Hardpoint randomized"). As long as there is some clear indication on the UI which sort of Hardpoint you are playing. I don't want someone to join a Randomized Hardpoint mid-round, think it's the original Hardpoint, then try to build a bunker on the "next" point only to find out that point actually will not happen (or, maybe already happened?) during the round. And vice versa, of course.

**Comment by Haox09** [comment]
All hardpoint should be changed and leave no seperate gamemode

**Comment by deferred-impact** [comment]
:moyai:

**Comment by Gregory-Y** [comment]
I agree with this suggestion, would make games less predictable and force people to adapt more which is good.
